build-bin/dist_to_gh_pages

```diff
@@ -15,26 +15,37 @@
 
 set -ue
 
+ORIGIN_BRANCH='master'
+TARGET_BRANCH='gh-pages'
+
 # Update gh-pages
-git fetch origin gh-pages:gh-pages
-git checkout gh-pages
+git fetch origin $TARGET_BRANCH:$TARGET_BRANCH
+git checkout $TARGET_BRANCH
+
+
+
+git checkout -qf $TARGET_BRANCH
+
+# Remove everything except .git
+rm -rf *
+
+git checkout $ORIGIN_BRANCH docs
```

Comment:
The less vocabulary in a script, the better. I was wondering what this "docs" directory was about. Consider the name of the repo is zipkin-api and there is no directory named docs anywhere. I was looking at this script later and wondering if something made a directory named docs via the build process.

In our docker builds, we use the directory named "install" that works, workspace, etc.

```suggestion
git checkout $DEFAULT_BRANCH workspace
```